<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>711</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>0</y>
      <width>129</width>
      <height>145</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/iitblogo.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>8</y>
      <width>209</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/Tufts_University_logo.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="PlotWidget" name="dyn_plot">
    <property name="geometry">
     <rect>
      <x>376</x>
      <y>144</y>
      <width>673</width>
      <height>169</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>144</y>
      <width>336</width>
      <height>345</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="tia_fb_res_lb">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>TIA feedback resistance</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="tia_fb_res_cb">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <item>
          <property name="text">
           <string>External resistance</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2.75k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3.5k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>35k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>120k</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>350k</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="rload_lb">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Rload (Ohms)     </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="rload_cb">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>33</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>50</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>100</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="ref_vol_src_lb">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Reference voltage source</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ref_vol_src_cb">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Internal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>External</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="int_zero_lb">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Internal Zero         </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="int_zero_cb">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <item>
          <property name="text">
           <string>660 mV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.65 V</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2.21 V</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3.3 V</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="bias_pol_lb">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Bias Polarity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="bias_pol_cb">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Negative (WE - RE) &lt; 0V</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Positive (WE - RE) &gt; 0V</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="bias_pol_lb_2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Bias (Percentage of the
          source reference)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="bias_cb">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <item>
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>33 mV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>66 mV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>132 mV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>198 mV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>264 mV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>330 mV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>396 mV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>462 mV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>528 mV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>594 mV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>660 mV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>726 mV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>792 mV</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="fet_short_lb">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>FET Short</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="fet_short_cb">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Disabled</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Enabled</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="mode_op_lb">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Mode of Operation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="mode_op_cb">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Deep Sleep</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2-lead ground ref galvanic cell</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Standby</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3-lead amperometric cell</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Temp measurement (TIA OFF)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Temp measurement (TIA ON)</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="configure_tag">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Configure</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>568</y>
      <width>529</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>91</number>
     </property>
     <item>
      <widget class="QPushButton" name="start_tag">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stop_tag">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quit_tag">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="lmp_bd_tag">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>504</y>
      <width>185</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>LMP91000 Block Diagram</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>544</y>
      <width>249</width>
      <height>49</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Time elapsed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="time_tag">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="mark_tag">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>616</y>
      <width>115</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Mark</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="marker_edit">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>608</y>
      <width>249</width>
      <height>49</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="dyn_plot2">
    <property name="geometry">
     <rect>
      <x>376</x>
      <y>336</y>
      <width>673</width>
      <height>169</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="tia_fb_res_lb_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>48</y>
      <width>425</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Data Acquisition Interface</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1064</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="IITB.qrc"/>
 </resources>
 <connections/>
</ui>
